spring-cloud-stream/src/main/java/org/springframework/cloud/stream/aggregate/AggregateApplication.java

```diff
@@ -92,12 +96,15 @@ private static void createChildContexts(ConfigurableApplicationContext parentCon
 			Class<?>[] apps, String args[][]) {
 		for (int i = apps.length - 1; i >= 0; i--) {
 			String appClassName = apps[i].getName();
-			embedApp(parentContext, getNamespace(appClassName, i), apps[i])
+			embedApp(parentContext, getNamespace(null, appClassName, i), apps[i])
```

Comment:
Based on [this] (https://github.com/spring-cloud/spring-cloud-stream/pull/452/files#r58666648), we can avoid passing null here.